sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+PREFIX dc:      <http://purl.org/dc/terms/>
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
+
+<http://www.w3.org/ns/sparql#> a owl:Ontology ;
+     dc:date "2025-06-30" ;
+     dc:description "Terms used by SPARQL 1.2" ;
+     owl:versionInfo ""
+     .
+
+##
+
+sparql:Function rdf:type rdfs:CLass .
+sparql:Operator rdf:type rdfs:CLass .
+sparql:FunctionalForm rdf:type rdfs:CLass .
+
+## @@ToDo
+
+## Operators
+
+# Arithmetic: +, -, *, /
+# Comparision: = , !=, >, >=, <, <=
+# Unary: !, +, -
+# Boolean: && and ||
+# Aggregates:          COUNT, SUM, MIN, MAX, AVG, SAMPLE, GROUP_CONCAT
+# Aggregates distinct: COUNT, SUM, MIN, MAX, AVG, SAMPLE, GROUP_CONCAT
+# Check WRT BuiltInCall
+
+##@@
+
+## Operators
+
+sparql:plus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:subtract rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:multiple rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:divide rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:unary-minus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:unary-plus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+# Section: Functional Forms
+
+sparql:bound rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-bound> ;
+    .
+
+sparql:if rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-if> ;
+    .
+
+sparql:coalesce rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-coalesce> ;
+    .
+
+sparql:filter-exists rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-filter-exists> ;
+    .
+
+sparql:logical-or rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-logical-or> ;
+    .
+
+sparql:logical-and rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-logical-and> ;
+    .
+
+sparql:in rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-in> ;
+    .
+
+sparql:not-in rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-not-in> ;
+    .
+
+# Section: Functions on RDF Terms
+
+sparql:sameTerm rdf:type sparql:Function ;
+    dc:description "" ;
```

Comment:
This function checks whether two RDF terms are the same in the strict sense, including same lexical form, datatype, and language tag for literals.

Reply:
```suggestion
    dc:description "This function checks whether two RDF terms are the same in the strict sense, including their lexical forms, datatypes, and language tags for literals." ;
```